module/importer/src/Domain/Command/Import/Attribute/ImportUpdateAttributesInProductCommand.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * Copyright © Ergonode Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Importer\Domain\Command\Import\Attribute;
+
+use Ergonode\Core\Domain\ValueObject\TranslatableString;
+use Ergonode\Importer\Domain\Command\ImporterCommandInterface;
+use Ergonode\Product\Domain\ValueObject\Sku;
+use Ergonode\SharedKernel\Domain\Aggregate\ImportId;
+
+class ImportUpdateAttributesInProductCommand implements ImporterCommandInterface
+{
+    private ImportId $importId;
+
+    /**
+     * @var TranslatableString[]
+     */
+    private array $attributes;
+
+    private Sku $sku;
+
+    /**
+     * @param TranslatableString[] $attributes
+     */
+    public function __construct(ImportId $importId, array $attributes, Sku $sku)
```

Comment:
Why is no ImportLineId property, its not necessary ?
Can I use this command i my custom import ???